#CAN-device parameters
CanDevice: /dev/pcan32
CanBaudrateVal: 1000
ModId: 1
#
#Joint parameters
JointName: seilwinde
EnoderIncrementsPerRevMot: 8000
MotorDirection: 1
GearRatio: 62.5
MaxVelRadS: 1
#
#Winch specific parameters
WinchDiameter: 0.10
